auth-api/migrations/versions/304705f06971_tasks_multiple_reasons.py

```diff
@@ -0,0 +1,84 @@
+"""tasks_multiple_reasons
+
+Revision ID: 304705f06971
+Revises: 41ac4080a7ed
+Create Date: 2021-09-10 14:50:36.964423
+
+"""
+from typing import List
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy import text
+
+from auth_api.models import Task
+
+# revision identifiers, used by Alembic.
+revision = '304705f06971'
+down_revision = '41ac4080a7ed'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # We get the list of tasks at first.
+    # we convert tasks.remarks from character type to array form ( check if there are arrays already, edge case )
+    # we convert the column remarks type from character to ARRAY
+
+    conn = op.get_bind()
+    task_res = conn.execute("SELECT * FROM tasks;")
+    task_list: List[Task] = task_res.fetchall()
+
+    for task in task_list:
+        if task.remarks is None:
+            continue
+        # A check if there are task remarks that is already wrapped in '{}', ideally, we would not hit this scenario
+        if task.remarks.startswith('{') and task.remarks.endswith('}'):
```

Comment:
you can remove the whole thing rite? its an upgrade script ;so we will never have this scenario
also select * from can check for tasks which remarks exists

Reply:
yeah refactored the condition now